src/util.ts

```diff
@@ -421,3 +421,35 @@ interface IUploadResult {
     mxcUrl: string;
     size: number;
 }
+
+// Type type
+type Type = Function;  // tslint:disable-line ban-types
+
+/**
+ * Returns true if `obj` is subtype of at least one of the given types.
+ */
+export function instanceofsome(obj: object, types: Type[]): boolean {
```

Comment:
can't you make these public static functions of the Util class like all the other ones? Then we stay consistent with `Util.wrap()` and `Util.instanceofsome()`

Reply:
Oh indeed, I didn't conform to consistency there.

Why is this Util class there at all btw? These functions have their own util module, so they are already wrapped nicely. This does seem to only add noise to the code using this methods, or is there some JS quirk which makes it necessary?